docassemble/juvenilesealing/data/questions/juvenile-sealing-intro-10.yml

```diff
@@ -35,18 +35,24 @@ code: |
       not_finishing_end
   else:
     progressor.increment()  # 1
-  
+
   progressor.increment()  # 2
-  
+
   needs_dockets = False
-  if ( not have_docket_copies_already is True ):
+  # If user doesn't answer yes for docket sheets, show user the docket sheet description
+  if ( not have_docket_copies_already ):
     progressor.increment()  # 3
     needs_dockets = True
-    if ( have_docket_copies_already is None ):  # They don't know
-      about_docket_sheets
-    if ( wants_dockets_first is True ):
-      progressor.increment()  # 4
-      not_finishing_end  # end
+    # If user wants to see docket sheet instructions
+    if ( wants_dockets_first ):
+      # User has not seen docket sheet instructions yet
+      if ( not request_docket_sheets ):
+        certified_copies_page_text_question
```

Comment:
What @berit says about it being the same question is true, but just for information for folks:

An id cannot be used to trigger a question. Only field names can be used. `certified_copies_page_text_question` is an id and would, I believe, create an error.

Reply:
Got it! I'll make this change.